src/test/java/api/exercise/Exercise1.java

```diff
@@ -0,0 +1,111 @@
+package api.exercise;
+
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import static org.junit.Assert.assertEquals;
+
+@SuppressWarnings({"ConstantConditions", "unused", "MismatchedQueryAndUpdateOfCollection"})
+public class Exercise1 {
+
+    enum Status {
+        UNKNOWN,
+        PENDING,
+        COMMENTED,
+        ACCEPTED,
+        DECLINED
+    }
+
+    @Test
+    public void acceptGreaterThan21OthersDecline() {
+        Person alex = new Person("Алексей", "Мельников", 20);
+        Person ivan = new Person("Иван", "Стрельцов", 24);
+        Person helen = new Person("Елена", "Рощина", 22);
+        Map<Person, Status> candidates = new HashMap<>();
+        candidates.put(alex, Status.PENDING);
+        candidates.put(ivan, Status.PENDING);
+        candidates.put(helen, Status.PENDING);
+
+        // TODO реализация
+        candidates.replaceAll((person,status) -> status = (person.getAge() > 21 ? Status.ACCEPTED : Status.DECLINED));
```

Comment:
There is no need to perform ```status = ``` because anyway returning value is a new value for the map.